src/MiddlewareListener.php

```diff
@@ -1,25 +1,26 @@
 <?php
 /**
- * @link      http://github.com/zendframework/zend-mvc for the canonical source repository
- * @copyright Copyright (c) 2005-2018 Zend Technologies USA Inc. (http://www.zend.com)
+ * @see       https://github.com/zendframework/zend-mvc for the canonical source repository
```

Comment:
Please use a separate commit for this changes. Don't mix different topics in one commit.

Reply:
I've separated the commits and force-pushed them.